js/researches/german/determinePassives.js

```diff
@@ -0,0 +1,29 @@
+var arrayToRegex = require( "../../stringProcessing/createRegexFromArray.js" );
+var auxiliaries = require( "./passivevoice-german/auxiliaries.js" )();
+var getParticiples = require( "./passivevoice-german/getParticiples.js" );
+
+var forEach = require( "lodash/forEach" );
+
+var auxiliaryRegex = arrayToRegex( auxiliaries );
+
+/**
+ * Determines whether a sentence part is passive.
+ *
+ * @param {string} sentencePartText The sentence part to determine voice for.
+ * @param {Array} auxiliaries A list with auxiliaries in this sentence part.
+ * @returns {boolean} Returns true if passive, otherwise returns false.
+ */
+module.exports = function( sentencePartText, auxiliaries ) {
+	var passive = false;
+	var auxiliaryMatches = sentencePartText.match( auxiliaryRegex );
+	if ( auxiliaryMatches === null ) {
+		return passive;
+	}
+	var participles = getParticiples( sentencePartText, auxiliaries );
```

Comment:
The code of lines 22-27 is exactly the same as done [in the English file](https://github.com/Yoast/YoastSEO.js/blob/43e653922cc30aa8c37f50f4af34479e73ca49ff/js/researches/english/determinePassives.js#L13-L19). Maybe it's better to make an util/helper/something for it.